.github/workflows/issue_validator.yml

```diff
@@ -30,6 +30,7 @@ jobs:
             }
           reproduction-comment: '.github/invalid-link.md'
           reproduction-hosts: 'github.com,codesandbox.io'
+          reproduction-blocklist: 'github.com/vercel/next.js.*'
```

Comment:
Let me know if this is desired or we have other patterns we want to include on this list